test/checkout.int.t.sol

```diff
@@ -455,7 +364,7 @@ contract Checkout_buyingFromTenRemovals_singleSupplier is Checkout {
   }
 }
 
-contract Checkout_buyingFromTenRemovals_singleSupplier_byApproval is Checkout {
```

Comment:
Can't tell from this diff, but did this permit. Asked test just get moved elsewhere?

Reply:
Same question for the previous tests listed that the diff appears to indicate were deleted

Reply:
there are a few tests that were removed because our contracts' functionality was actually reduced. for example, you can't buy from a single supplier with a permit anymore -- we removed that endpoint because it's not public and now it's part of the special order flow which is all approval based.  This particular test was just renamed I believe to drop the `byApproval` because there's no longer a permit v.s. approval distinction for specific supplier purchases.  I didn't delete any tests I could rewrite to accommodate the new contract interface -- no test functionality should have been reduced except for cases where our contract's functionality is actually being reduced and the test no longer applies and couldn't be converted to an analogous test using the new special order swap function.